x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/ForceMergeStepTests.java

```diff
@@ -53,18 +58,29 @@ public ForceMergeStep mutateInstance(ForceMergeStep instance) {
                 nextKey = new StepKey(key.getPhase(), key.getAction(), key.getName() + randomAlphaOfLength(5));
                 break;
             case 2:
-                maxNumSegments += 1;
+                onlyExpungeDeletes = onlyExpungeDeletes == false;
+                if (onlyExpungeDeletes) {
+                    maxNumSegments = null;
+                } else {
+                    maxNumSegments = randomIntBetween(1, 10);
```

Comment:
Shall we keep `maxNumSegments += 1;` here to avoid a clash? Or otherwise use `randomValueOtherThan`